batteries/difftext/init.luau

```diff
@@ -0,0 +1,33 @@
+local myersdiff = require("./difftext/myersdiff")
+local printdiff = require("./difftext/printdiff")
+
+export type Diff = myersdiff.Diff
+export type ApplyDiff = (string) -> string
+
+local function diff(
+	a: string,
+	b: string,
+	options: {
+		byChar: boolean?,
+	}?
+): (Diff, ApplyDiff)
```

Comment:
why do we need applydiff too? if it's just for the tests can't we just check the diff structurally?